core/transcoder.go

```diff
@@ -91,6 +150,28 @@ func (nv *NvidiaTranscoder) Stop() {
 	nv.session.StopTranscoder()
 }
 
+func runTranscodeLoop(gpu string, stack *segStack, workDir string) {
+	for {
+		seg := stack.pop()
+		// Set up in / out config
+		in := &ffmpeg.TranscodeOptionsIn{
+			Fname:  seg.fname,
+			Accel:  ffmpeg.Nvidia,
+			Device: gpu,
+		}
+		opts := profilesToTranscodeOptions(workDir, ffmpeg.Nvidia, seg.profiles)
+		// Do the Transcoding
+		res, err := seg.session.Transcode(in, opts)
+		if err != nil {
+			seg.res <- &nvSegResult{nil, err}
+			continue
+		}
+		td, err := resToTranscodeData(res, opts)
+		seg.res <- &nvSegResult{td, err}
+		continue
```

Comment:
Looks like this `continue` is unnecessary.

Reply:
Fixed in 26c1f19b46bb9546bdbce22cbe3f890d957e0310